<launch>

    <include file="$(find realsense2_camera)/launch/rs_camera.launch" />  

    <node pkg="hebi_rosie_demo" type="vision_threshold" name="vision_process" output="screen" />

  </launch>
